theme6/source/stylesheets/site.css.scss

```diff
@@ -1,24 +1,159 @@
-body {
-  background-color: #fbc547;
-  color: #333;
-  font-family: -apple-system, BlinkMacSystemFont, "Avenir Next", "Avenir",
-    "Segoe UI", "Lucida Grande", "Helvetica Neue", "Helvetica", "Fira Sans",
-    "Roboto", "Noto", "Droid Sans", "Cantarell", "Oxygen", "Ubuntu",
-    "Franklin Gothic Medium", "Century Gothic", "Liberation Sans", sans-serif;
-  padding: 18vh 1rem;
-  text-align: center;
-}
-
-a {
-  color: rgba(#000, 0.7);
-
-  &:focus,
-  &:hover {
-    color: rgba(#000, 0.6);
-  }
-}
-
-.middleman-logo {
-  margin-bottom: 1rem;
-  width: 10rem;
-}
+
+
+/*==========Stylesheet DESKTOP========*/
+    footer {
+      bottom: 0;
+      display: inline;
+      width: 100%;
+      
+    }
+
+    .information{justify-content: center;}
+
+    li{display: inline;}
```

Comment:
If you are going to keep it in one line add spacing to it like so: `li { display: inline; }`